scenarios/crowdsecurity/CVE-2022-46169.yaml

```diff
@@ -0,0 +1,29 @@
+type: leaky
+name: crowdsecurity/CVE-2022-46169-bf
+description: "Detect CVE-2022-46169 brute forcing"
+filter: |
+    Upper(evt.Meta.http_path) contains Upper('/remote_agent.php') &&
+    Upper(evt.Parsed.verb) == 'GET' &&
+    Lower(evt.Parsed.http_args) contains 'host_id' &&
+    Lower(evt.Parsed.http_args) contains 'local_data_ids'
+leakspeed: "10s"
+capacity: 5
+blackhole: 1m
+groupby: "evt.Meta.source_ip"
+labels:
+  type: exploit
+  remediation: true
+---
+type: trigger
+name: crowdsecurity/CVE-2022-46169-cmd
+description: "Detect CVE-2022-46169 cmd injection"
+filter: |
+    Upper(evt.Meta.http_path) contains Upper('/remote_agent.php') &&
+    Upper(evt.Parsed.verb) == 'GET' &&
+    Lower(evt.Parsed.http_args) contains 'action=polldata' &&
+    (Lower(evt.Parsed.http_args) contains 'poller_id=;' || Lower(evt.Parsed.http_args) contains 'poller_id=%3b')
```

Comment:
Could be bypassed if `;` is not directly at front? maybe somebody with better regex knowledge can do one?